ERCS/erc-7662.md

```diff
@@ -0,0 +1,273 @@
+---
+eip: 7662
```

Comment:
```suggestion
eip: 7656
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.